readthedocs/doc_builder/python_environments.py

```diff
@@ -224,7 +224,7 @@ def install_core_requirements(self):
             self.project.get_feature_value(
                 Feature.USE_SETUPTOOLS_LATEST,
                 positive='setuptools<40',
```

Comment:
Regarding your question, I'm +1 on pinning `<41` on positive so that we're testing as soon as a new backwards incompat package is out.

Reply:
If we pin `<41` here, what should we do with the 10k project that have already the feature flag? Should we remove this feature from those or should we keep as they are?

It won't be a problem immediately, but it could be an issue when 40 is released... maybe.